client/app/components/accounts/wizard/creation.cjsx

```diff
@@ -0,0 +1,186 @@
+{IMAP_OPTIONS} = require '../../../constants/defaults'
+{Requests
+RequestStatus
+OAuthDomains} = require '../../../constants/app_constants'
+
+_     = require 'underscore'
+React = require 'react'
+
+Form    = require '../../basics/form'
+Servers = require '../servers'
+
+RequestsInFlightStore = require '../../../stores/requests_in_flight_store'
+
+RouterGetter = require '../../../getters/router'
+
+AccountActionCreator = require '../../../actions/account_action_creator'
+
+StoreWatchMixin = require '../../../mixins/store_watch_mixin'
+AccountMixin    = require '../../../mixins/account_mixin'
+
+
+ALERTS =
+    'DISCOVER_FAILED': 'DISCOVER_FAILED'
+    'CHECK_FAILED':    'CHECK_FAILED'
+    'CREATE_FAILED':   'CREATE_FAILED'
+
+
+module.exports = AccountWizardCreation = React.createClass
+
+    displayName: 'AccountWizardCreation'
+
+    mixins: [
+        StoreWatchMixin [RequestsInFlightStore]
+        AccountMixin
```

Comment:
Are _mixins_ as bad as we suggest ;)? In this case, I'm curious about some feedback: we'lle have two top level components: _AccountCreation_ and _AccountEdit_. Both exposes a `<Server>` component and manage an _Account_ data from _AccountStore_. They 're similar but not strictly the sames. We need a way to share some logics, taht are pure data parsing (and that never _get_ some information from a store nor anywhere). Plus, we need the same parsing logic in some _ActionCreator_. I think _mixins_ are trhe best way to handle this use case. What's your suggestion?